barretenberg/cpp/src/barretenberg/relations/ecc_op_queue_relation.hpp

```diff
@@ -20,16 +20,15 @@ template <typename FF_> class EccOpQueueRelationImpl {
 
     template <typename AllEntities> inline static bool skip([[maybe_unused]] const AllEntities& in)
     {
-        // The prover can skip execution of this relation altogether since an honest input will lead to a zero
-        // contribution at every row, even when the selector lagrange_ecc_op is on
-        return true;
```

Comment:
i have thought at some point that this is bizzare but didn't get the chance to investigate it

Reply:
yeah I think this was just faulty/rushed thinking that got reinforced by an erroneously passing test